packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -251,7 +256,24 @@ const prompts = () => {
         // NOTE: there's no question about Einstein's _site_ id because we currently assume that the site id will be the same for both Commerce API and Einstein
     ]
 
-    return inquirer.prompt(questions).then((answers) => buildAnswers(answers))
+    const questions = [
+        {
+            name: 'useDemoSettings',
+            message: 'Do you want to try out PWA Kit with the demo storefront?',
+            choices: [demoChoice, customChoice],
+            type: 'list'
+        },
+        ...customChoiceQuestions.map((question) => ({
+            ...question,
+            when: (answers) => answers.useDemoSettings == customChoice
+        }))
+    ]
```

Comment:
In my view this is a convoluted way of constructing these questions. You're generating a big long list of questions and then hiding them it is extremely difficult to see at a glance what the questions will be, when using a given preset.

Please let me suggest an alternative before you merge.

Reply:
Sure, I'll hold off on merging this for now.

Reply:
Can we resolve this issue now @olibrook?